### PR TITLE
Documentation updated. (For 0.2.0 version)

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

New version of documentation (swagger).
This update include swagger 3 and the new vision of Hermod conception.

[Swagger UI available here](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Hermod/swagger/swagger-spec.yaml)

### CODE REVIEW

File: swagger-spec.yaml [hunk 1]
```diff
@@ -1,234 +1,286 @@
...
+                properties:
+                  status:
+                    type: "string"
+                    enum: ["OK", "KO"]
+components:
+  securitySchemes:
+    apiKey:
+      type: apiKey
+      in: header
+      name: Autorization
```
Comment (vincentlepot) [inline swagger-spec.yaml]: Aut**h**orization

Comment (vincentlepot) [reply swagger-spec.yaml]: In fact, it's a http basic auth

Comment (Netmisa) [reply swagger-spec.yaml]: See with @vincentlepot we keep `apiKey` type

**Comment by Netmisa** [comment]
Move to `release_0.2 branch` https://github.com/CanalTP/Hermod/pull/32

**Pull request closed without merging**